src/validation.cpp

```diff
@@ -4282,17 +4294,6 @@ bool LoadGenesisBlock(const CChainParams& chainparams)
     if (mapBlockIndex.count(chainparams.GenesisBlock().GetHash()))
         return true;
 
-    // Use the provided setting for -addressindex in the new database
```

Comment:
Does `LoadBlockIndex` get called anyway so that this section is not needed / redundant?

Reply:
This is basically the same logic as for txindex which was moved earlier https://github.com/dashpay/dash/pull/3096/commits/dbd49930849b44d9b200df088006ef70cc0807e2#diff-24efdb00bfbe56b140fb006b562cc70bR4182 (see the description of the commit or original PR we backported there).